main.go

```diff
@@ -72,6 +72,7 @@ func handleTarget(w http.ResponseWriter, req *http.Request) {
 	configMutex.Lock()
 	registry := prometheus.NewRegistry()
 	collector, err := NewCwCollector(target, task, region)
+
```

Comment:
Remove this new line change ( it means there are no changes on main.go )

Reply:
Done :)